SPECS/linuxptp/linuxptp.spec

```diff
@@ -107,10 +130,15 @@ PATH=..:$PATH ./run
 %{_sbindir}/ptp4l
 %{_sbindir}/timemaster
 %{_sbindir}/ts2phc
+%{_sbindir}/tz2alt
 %{_mandir}/man5/*.5*
 %{_mandir}/man8/*.8*
 
 %changelog
+* Wed Jun 04 2025 Aaron Chan <aaron.chun.yew.chan@intel.com> - 4.3-1
+- Add TSN patches/support
+- update to 4.3
```

Comment:
Please add "Initial Edge Microvisor import ..."

Reply:
Ok let me add this in

Reply:
Updated changelog to initial import to EMT3